test/client/fixtures/sandbox/event/focus-blur-change-test.js

```diff
@@ -770,12 +770,22 @@ if (window.HTMLInputElement.prototype.setSelectionRange) {
 
                 input2.setSelectionRange(0, 0);
 
-                window.setTimeout(function () {
-                    strictEqual(focusRaised, needFocus);
-                    strictEqual(document.activeElement === input2, needFocus);
+                var maxAttemptCount = 5;
+                var attemptCount    = 0;
+
+                var intervalId = window.setInterval(function () {
+                    attemptCount++;
+                    var isInputFocused = document.activeElement === input2;
+
+                    if (focusRaised === needFocus && isInputFocused === needFocus ) {
+                        expect(0);
+                        window.clearInterval(intervalId);
+                        startNext();
+                    }
+                    else if (attemptCount > maxAttemptCount)
+                        ok(false, 'The check for calling focus failed');
```

Comment:
The check actually succeeds - it's the result that is unexpected, so maybe:
'The element's focus state doesn't match the expected one.'